### PR TITLE
cricklewood new site

### PR DESCRIPTION
**Pull request by natesheehan**
# Cricklewood
---
This [Site](https://www.northwesttwo.org.uk/bq-site-may2021/) has plans for 1100 flats to be built in Cricklewood, Brent, London.  The site was requested through [twitter](https://twitter.com/CricklewoodMum/status/1394983348422991873).

### Notes:
Due to an issue with [cyclestreets](https://github.com/Robinlovelace/cyclestreets/issues/2), the actdev model is unable to calculate the shortest route to town (in this case, because it is too small, as the site is directly in the centre of the neighbourhood). For hot-fix purposes, a new town centre of north Cricklewood has been chosen.

All sanity checks are passing ✔️

### CODE REVIEW

**Review by Robinlovelace — COMMENTED** [review]
Site location looks to match another I was sent:

image

image

File: data-small/all-sites.csv [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
-tyersal-lane,Tyersal Lane,Leeds,partly,270,https://publicaccess.leeds.gov.uk/online-applications/applicationDetails.do?keyVal=NRFIZQJB17S00&activeTab=summary,14,-5,-21,14,5,3.5,4,13,1,59,15,2,10,2.54,1.7,3.89,2.61,2.68,3.08,60,14,0,green,red,amber
+tyersal-lane,Tyersal Lane,Bradford,partly,270,https://publicaccess.leeds.gov.uk/online-applications/applicationDetails.do?keyVal=NRFIZQJB17S00&activeTab=summary,14,-5,-21,14,5,3.5,4,13,1,59,15,2,10,2.54,1.7,3.89,2.61,2.68,3.08,60,14,0,green,red,amber
 upton,Upton,Northampton,mostly (partly before 2011),1382,NA,4,31,50,32,6.8,4.3,3,3,2,83,3,3,6,1.3,1.31,2.3,1.42,1.45,1.58,56,6,25,red,amber,red
-water-lane,Water Lane (Livable Exeter),Mid Devon,no,1567,https://www.liveableexeter.co.uk/garden-communities/garden-communities/water-lane/,30,52,63,58,1.8,1.4,1,22,8,53,9,1,7,1.59,1.35,4.13,NA,NA,NA,25,34,24,NA,NA,NA
+water-lane,Water Lane (Livable Exeter),Exeter,no,1567,https://www.liveableexeter.co.uk/garden-communities/garden-communities/water-lane/,30,52,63,58,1.8,1.4,1,22,8,53,9,1,7,1.59,1.35,4.13,NA,NA,NA,25,34,24,NA,NA,NA
```
Comment (Robinlovelace) [inline data-small/all-sites.csv:42]: Looks better to me.

**Review by Robinlovelace — APPROVED** [review]
Looks good. If you could tidy up .png files in the route directory of the repo like https://github.com/cyipt/actdev/pull/173/files#diff-b200dbfc962f81245ef2e4a61b6276f785dbbf39836c3d4a6405e236dfcc2310 pre or post merge that would be great. Otherwise this is good to go!

**Comment by natesheehan** [comment]
Crinklewood site is live on [actdev-ui](https://actdev.cyipt.bike/cricklewood/#5.82/53.035/-0.763) 🚀 

@dabreegster ab street sim files will be in https://github.com/cyipt/actdev/tree/main/data-small/cricklewood

**Pull request merged**